plugins/lookup/merge_variables.py

```diff
@@ -61,6 +61,27 @@
         ini:
           - section: merge_variables_lookup
             key: override
+      all_hosts:
+        description:
+          - Search for variables accross all hosts. This mode allows to collect configuration pieces
+             accross different hosts (e.g. a service on a host with its database on another host).
+          - Each of the matching variables may container a property which specified for which host(s)
+             it is intended (see parameter 'target_host_property').
+          - If that property isn't included in the matching variable, it only applies for the same host.
+        type: bool
+        default: false
+        choices:
+          - True
+          - False
```

Comment:
Booleans have two possible values, there is no need to provide `choices`. You need to provide `version_added` though:
```suggestion
        version_added: 8.4.0
```